From 36ff885bc7d7d906ce63f29dd04e9364a37590da Mon Sep 17 00:00:00 2001
From: "Eric S. Raymond" <esr@thyrsus.com>
Date: Thu, 10 May 2012 17:20:15 -0400
Subject: [PATCH] Coverity complains because the isdst field isn't
 initialized.

Probably not an actual defect given the semantics of timegm(3), but
let's be safe.
---
 driver_oncore.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/driver_oncore.c b/driver_oncore.c
index 6ccb073..b020c7f 100644
--- a/driver_oncore.c
+++ b/driver_oncore.c
@@ -109,6 +109,7 @@ oncore_msg_navsol(struct gps_device_t *session, unsigned char *buf,
 	unpacked_date.tm_hour = (int)getub(buf, 8);
 	unpacked_date.tm_min = (int)getub(buf, 9);
 	unpacked_date.tm_sec = (int)getub(buf, 10);
+	unpacked_date.tm_isdst = 0;
 	nsec = (uint) getbeu32(buf, 11);
 
 	/*@ -unrecog */
-- 
1.7.10

